MobileCenterCrashes/MobileCenterCrashes/Internals/Model/MSAppleErrorLog.m

```diff
@@ -86,6 +86,9 @@ - (BOOL)isValid {
 - (BOOL)isEqual:(MSAppleErrorLog *)errorLog {
   if (!errorLog)
     return NO;
+  
+  if (![super isEqual:errorLog])
```

Comment:
@TroubleMakerBen to compare `MSAbstractErrorLog` properties. Without this instances with different errorId/processId/... considered equal.